src/middleware/src/storage/src/websql.js

```diff
@@ -7,20 +7,20 @@ import isFunction from 'lodash/isFunction';
 import isString from 'lodash/isString';
 const idAttribute = process.env.KINVEY_ID_ATTRIBUTE || '_id';
 const masterCollectionName = 'sqlite_master';
-const size = 5 * 1000 * 1000; // Database size in bytes
 let dbCache = {};
-let isSupported = undefined;
+let isSupported;
 
 export default class WebSQL {
-  constructor(name = 'kinvey') {
+  constructor(name = 'kinvey', size = 5000) {
```

Comment:
Having it parameterized is fine, but I didn't find the default easily. The way it was written previously (as a `const`) allowed me to quickly identify the default value.